app/components/confirmation-modal.hbs

```diff
@@ -1,11 +1,9 @@
-{{! TODO: This model should be renamed from "inactive" to "non-active". Or it should be merged with the other confirmation modal into a single general confirmation modal.}}
 <AuModal @modalOpen={{true}} @closeModal={{@onCancel}}>
   <:title>Bevestig</:title>
-  <:body>Opgelet, je hebt een niet actieve organisatie geselecteerd. Ben je
-    zeker dat je deze relatie wenst toe te voegen?
-  </:body>
+  <:body>{{@message}}</:body>
   <:footer>
-    <AuToolbar @reverse={{true}} as |Group|>
```

Comment:
Didn't this work as expected or something?

Reply:
the `@reverse` argument was removed and replaced again with an empty `<Group />` so I assumed things maybe didn't work as expected (and there would be a bug in Appuniversum).

Reply:
Wait, now it's fine :smile: maybe I was looking at an outdated page when I posted that, sorry for the noise.